amaznot/routes/web.php

```diff
@@ -60,4 +61,7 @@
 Route::get('/orders/{id}', [OrderController::class, 'orderDetails'])->name('ordersitems');
 Route::get('/orders/{order_id}/{product_id}', [OrderController::class, 'productDetails'])->name('product');
 Route::post('/orders/deleteOrder', [OrderController::class, 'deleteOrder'])->name('deleteOrder');
-Route::post('/orders', [OrderController::class, 'store']);
\ No newline at end of file
+Route::post('/orders', [OrderController::class, 'store']);
+
+// Search Functions
+Route::get('/search/{search?}', [SearchController::class, 'search'])->name('search');
```

Comment:
Is the ? after the word search supposed to be there?

Reply:
Yeah, I'll send you the link, so u can fully confirm